### ISSUE TITLE
AMP Stories: What's the recommended aspect ratio of portrait images to be used in the fill layout?

### ISSUE THREAD

**Issue by AutadeJS** [ampproject/amphtml#15920]
## What's the issue?
There's no clarity about aspect ratio of portrait images to be used in AMP Stories. Some help pages recommend 3x4 while others 9x16.

**Comment by AutadeJS** [comment]
960x1280 (3:4) vs 720x1280 (9:16) which image will FILL better on most mobiles, without cutting content on top?

**Comment by newmuis** [comment]
@AutadeJS do you have pointers to the two places that give conflicting information?

/to @hongwei1990 for a specific recommendation, but also note that this is likely to shift over time, as devices gain/lose market share.  The best practice is to ensure your design is at least somewhat responsive, and to allow for some amount of cropping on the sides of the assets.

**Comment by AutadeJS** [comment]
@newmuis Congratulations of launch of AMP Stories 1.0

On image sizes, what I mean is when we create images to use as fill background in a page in AMP stories, what size do we create them? What's your official recommendation?

**Comment by hongwei1990** [comment]
Sorry for the conflicting information that you saw in different places. Please let us know where you saw it so we can fix it. The recommended aspect ratio for vertical background content is 3:4. The viewport aspect ratio on standard devices without browser navigation and OS navigation is close to 17:25, so 3:4 scales better than 9:16.

**Comment by AutadeJS** [comment]
Thank you so much. I'll let you know soon where I saw the 720x1280.

**Comment by AutadeJS** [comment]
https://www.ampproject.org/docs/design/visual_story/create_cover_page
Uses 720x1280 for fill example.

https://www.ampproject.org/docs/design/amp_story_best_practices
This "AMP Story Best Practices" page says: 
The recommended dimensions for a poster image are: 720p (720w x 1280h) .
But it's for video poster.

**Comment by newmuis** [comment]
@bpaduch is it okay to change these examples to use 720x960 (720p at 3:4 aspect ratio)?

**Comment by bpaduch** [comment]
Sure, go ahead.  Make sure to update the file in the .zip too.

**Comment by AutadeJS** [comment]
@hongwei1990 says above "so 3:4 scales better than 9:16"

But this latest official video https://www.youtube.com/watch?v=2WjFBnCj2ew titled "Best practices for creating an AMP story" is again is giving 720x1280 as standard size. Shouldn't the recommended size be 960x1280?

Even the 2 help links above still recommend 720x1280.

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @newmuis Please triage this to an appropriate milestone.

**Comment by AutadeJS** [comment]
@newmuis 
What is the final recommended aspect ratio of portrait images to be used in Web Stories?

**Comment by newmuis** [comment]
We've settled on using 9:16 images, but expecting that they could be cropped to as short as 2:3.

We're working on providing documentation on this, as well as figuring out the easiest ways for creators to leverage this

**Comment by AutadeJS** [comment]
Ok, thank you.

**Comment by AutadeJS** [comment]
@newmuis 
There's some conflicting info in the documentation:

On https://amp.dev/documentation/components/amp-story/#poster-portrait-src-(required)
it's given:
**poster-portrait-src (required)
A URL to the story poster in portrait format (3x4 aspect ratio).**

That means if the cover slide's background image is 9:16 then it cannot be used as portrait because the required aspect ratio is 3:4

**Comment by AutadeJS** [comment]
On the same page, under **Poster guidelines** it's given:
Portrait: 640px x 853px
which is 3:4.

So is it ok to have Portrait as 9:16 to reuse the cover image as story portrait image?

**Comment by AutadeJS** [comment]
On https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/
it's said: **Use full screen, portrait images with good resolution (828 x 1,792).**
This is not 3:4

**Comment by newmuis** [comment]
@AutadeJS thanks for compiling this.  We can definitely improve this documentation.

Regarding `poster-portrait-src`, it can be a 3:4 crop of the 9:16 background image of the first page, if you like.  They should not literally reuse the same asset due to the differences in aspect ratio, as you've mentioned.

For the 828x1792 recommendation from https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/, this seems oddly specific, but also incorrect.  @hongwei1990 @gmajoulet @CrystalOnScript @tingyiwei do any of you know where this recommendation came from?

**Comment by AutadeJS** [comment]
It'd rather be easier to allow using the cover image itself as story's portrait. But that'd require to keep required aspect ratio of both same

**Comment by AutadeJS** [comment]
Apart from allowing user to upload a separate portrait, a story builder should also allow user to choose one of the page backgrounds as portrait. Especially the cover page's bg image should be the default portrait until user changes it. But different aspect ratio requirements are conflicting with this idea.

Can the portrait be 9:16 or it is AMP error to use that aspect ratio?

**Comment by AutadeJS** [comment]
If a website wants to create a library of images that can used as story posters as well as background images of pages then the aspect ratio will have to be same.

**Comment by newmuis** [comment]
Both of these features can still be accomplished by creating a 3:4 crop of the 9:16 asset.

Given the differences in mobile device sizing, it's also not guaranteed that the full 9:16 asset will be shown when actually viewing the story (in fact, that is rarely the case).  The guidance is to expect the asset to potentially be cropped as short as 2:3.

**Comment by AutadeJS** [comment]
Creating a 3:4 crop of the 9:16 asset: Yes that's the issue, making 2 copies of the image. Moreover asking user to crop an image twice will be UX issue.

**Comment by AutadeJS** [comment]
The requirement here is very simple: being able to create a library of images that can be reused as 

- cover page background 
- any other page background 
- story portrait. 

If such a library is possible the UI/UX can be made easy and overall productivity can go up.

But that requires the aspect ratio to be fixed at 9:16 (even though it isn't guaranteed but one still has to fixate on something).

The story portrait's different aspect ratio requirement is conflicting with all above. If the story portrait is one of the page's background itself then it's more correct.

**Comment by newmuis** [comment]
> Yes that's the issue, making 2 copies of the image. Moreover asking user to crop an image twice will be UX issue.

Two notes here:

1. It's possible for the story to be viewed at 3:4 as well.  So, the user would already need to specify what should happen if the asset were to be cropped to 3:4, right?
2. It's a different context.  It's not necessarily the case that using the same asset is the correct thing to do.

> The story portrait's different aspect ratio requirement of 3:4 is conflicting with all above. If the story portrait is one of the page's background itself then it's more correct.

I wouldn't frame it as being "more correct"; the poster image is representative of the story, and not literally tied to the first page.  From the documentation:

> - The poster image should be representative of the entire AMP story.
> - The poster image should be visible to the user when the AMP story begins. To accommodate sizing, cropping or minor styling changes or preview purposes, the image file URL used in the metadata does not need to be an exact match to the URL on the first page of the story.
> - Provide a raster file, such as .jpg, .png, or .gif. Avoid vector files, such as .svg or .eps.
> - The poster image should be in 3x4 aspect ratio for portrait, 4x3 for landscape, and 1x1 for square.
> - If the poster image is derived from a frame in a video, the thumbnail should be representative of the video. For example, the first frame in a video is often not representative.

These are, fundamentally, separate assets by design.  Simplifying the user experience for the case you've mentioned should be possible, but the poster image aspect ratios cannot be changed.   For one, it would be a breaking change (as platforms on both the creation and consumption sides have implemented against these existing ratios), but also, it's the more desirable aspect ratio on the consumption side from a UX perspective, since, for example, you can generally fit a nearly-full-bleed 3:4 card within the viewport, but cannot do so with a 9:16 card.

/cc @hongwei1990

**Comment by AutadeJS** [comment]
Agree with

- "it would be a breaking change (as platforms on both the creation and consumption sides have implemented against these existing ratios)"
- "it's the more desirable aspect ratio on the consumption side from a UX perspective"

**Comment by alankent** [comment]
This get finalized? I just noticed makestories.io released some tutorials where they recommend 4:3, 3:4, and 1:1. https://university.makestories.io/lesson/preparing-your-stories-for-publishing/ at 5:30 shows the poster size images they create (696x928, 928x696, and 696x696). It shows https://amp.dev/documentation/components/amp-story/#poster-portrait-src-(required) so I assume this is the final recommendations, but I came across this open ticket so was not sure.
